go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/onsi/gomega
+
+require (
+	github.com/fsnotify/fsnotify v1.4.7 // indirect
+	github.com/golang/protobuf v1.1.0
+	github.com/hpcloud/tail v1.0.0 // indirect
+	github.com/onsi/ginkgo v1.6.0
+	golang.org/x/net v0.0.0-20180724234803-3673e40ba225
```

Comment:
We generated this file as part of doing a review, and got slightly different different timestamps/checksums.  Would you mind updating the file so that it reflects and official go version, rather than a beta?

Reply:
Google has not yet tagged versions for golang.org/x/net, unfortunately.